recipes-multimedia/kodi/kodi_git.bb

```diff
@@ -164,6 +166,8 @@ do_configure:prepend() {
 	ln -sf $liblto ${STAGING_LIBDIR_NATIVE}/bfd-plugins/liblto_plugin.so
 
 	sed -i -e 's:CMAKE_NM}:}${TARGET_PREFIX}gcc-nm:' ${S}/xbmc/cores/DllLoader/exports/CMakeLists.txt
+	
```

Comment:
I think i didn't need to do this so far, because my waylandpp recipe builds wayland-scanner for target also. Not sure what would be the better approach